iceberg/iceberg-handler/src/test/queries/positive/iceberg_major_compaction_partition_evolution_ordered.q

```diff
@@ -60,10 +61,10 @@ select * from ice_orc where dept_id = 2 order by first_name;
 select * from ice_orc where dept_id = 3 order by first_name;
 describe formatted ice_orc;
 
-explain alter table ice_orc COMPACT 'major' and wait order by first_name desc;
+explain alter table ice_orc COMPACT 'smart_optimize' and wait order by first_name desc;
```

Comment:
we shouldn't support that, 'smart_optimize' should be implicit type for optimize command